developers_affiliations4.txt

```diff
@@ -18452,7 +18453,7 @@ hongbin: hongbin034!gmail.com
 hongchaodeng: fengjingchao!gmail.com, hongchaodeng!users.noreply.github.com, hongchaodeng1!gmail.com
 	Adobe Inc. until 2013-09-01
 	Cloudera Inc. from 2013-09-01 until 2015-11-01
-	Red Hat Inc. from 2015-11-01 until 2018-03-01
+	CoreOS Inc. from 2015-11-01 until 2018-03-01
```

Comment:
Was considering splitting out 1 month of work for RedHat, but forgot.

Reply:
SO I will do this in post merge, since 2/2/18 right?